pkg/frontend/validate.go

```diff
@@ -119,6 +119,25 @@ func validateAdminVMName(vmName string) error {
 	return nil
 }
 
+func validateAdminKubernetesPodLogs(name, namespace, containerName string) error {
```

Comment:
It is not important, but order of params is a bit odd: name then namespace and then container name. If it is not a lot of work - could you please reorder?

I think this will match order in `KubeGetPodLogs`. 
```suggestion
func validateAdminKubernetesPodLogs(namespace, name, containerName string) error {
```

Reply:
oky will do that..

Reply:
done